### ISSUE TITLE
Autosynth should label PRs appropriately

### ISSUE THREAD

**Issue by tseaver** [googleapis/synthtool#138]
At a minimum, it should apply the correct `api: *` label to the PR.  Even better would be to add the label which indicates the issue is autogen (in `google-cloud-python`, we use `codegen`).

**Comment by danoscarmike** [comment]
Could we extend this to issues as well as PRs? Particularly I'm interested to have the `api: {apiname}` label applied to 'Synthesis failed for {apiname}' issues.

**Comment by theacodes** [comment]
@danoscarmike yeah that should be trivial once the labeling of the PR is in place.

**Comment by busunkim96** [comment]
Autosynth now adds `api: *` labels to any PRs and issues it opens. (It will only do this on the mono-repos.)

I will open a separate issue for any other labels (like `codegen`) different languages would like to have applied to Autosynth PRs.

**Comment by SurferJeffAtGoogle** [comment]
The API name is now included in the pull request title for monorepos.  [Example PR](https://github.com/googleapis/google-cloud-ruby/pull/6270).
